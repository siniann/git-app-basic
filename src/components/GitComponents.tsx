import React from 'react';
import { Button } from '@material-ui/core';
import {
  gitStatus, gitClone, gitPush,
} from '../helper/gitFunctions';

const GitComponents = () => (
  <div>
    <Button onClick={gitClone}> Init git</Button>
    <Button onClick={gitStatus}> Check git status</Button>
    {/*
        <Button onClick={gitCommitLog}>View log</Button>
*/}
    <Button id="btn-push" onClick={gitPush}>Push to Git</Button>
  </div>
);

export default GitComponents;
