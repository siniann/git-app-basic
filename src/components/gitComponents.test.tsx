import React from 'react';
import { shallow } from 'enzyme';
import GitComponents from './GitComponents';

describe('Git components', () => {
  let mockFunction;
  let gitComponentShallow;
  beforeEach(() => {
    jest.clearAllMocks();
    mockFunction = jest.fn();
    gitComponentShallow = shallow(<GitComponents />);
  });

  it('should render properly ', () => {
    expect(gitComponentShallow).toMatchSnapshot();
  });
});
