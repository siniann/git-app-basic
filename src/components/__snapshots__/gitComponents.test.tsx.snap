// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Git components should render properly  1`] = `ShallowWrapper {}`;
